CHANGELOG.md

```diff
@@ -54,6 +54,10 @@ per cluster.
 - Remove tag: Application.
 - Prevent runtime baking, i.e. pcluster create-cluster only works for official AMIs or custom AMIs created by pcluster createami command.
 - Retain CloudWatch logs on cluster deletion by default. If you want to delete the logs during cluster deletion, set Monitoring > Logs > CloudWatch > RetainOnDeletion to False in the configuration file. 
+- Add multiple queues and compute resources support for pcluster configure when the scheduler is Slurm.
+- Add prompt for availability zone in pcluster configure automated subnets creation.
+- Remove support for CentOS8.
+- Remove support for CentOS7 on ARM.
```

Comment:
I think we never supported Centos7 on ARM so non need to mention in changelog

Reply:
Done